src/proxy-ui-api/frontend/src/views/Clients/Subsystem.vue

```diff
@@ -66,6 +91,25 @@ export default Vue.extend({
       );
     },
 
+    showDelete(): boolean {
+      if (!this.client) {
+        return false;
+      }
+
+      if (
+        this.client.status === 'REGISTERED' ||
+        this.client.status === 'REGISTRATION_IN_PROGRESS'
+      ) {
+        return false;
+      }
+
+      if (this.$store.getters.hasPermission(Permissions.SEND_CLIENT_DEL_REQ)) {
```

Comment:
`return this.$store.getters.hasPermission(Permissions.SEND_CLIENT_DEL_REQ)`

Reply:
Fixed